### ISSUE TITLE
Adding Liblouis Braille tables including Māori

### ISSUE THREAD

**Issue by ainekc** [nvaccess/nvda#15976]
### Is your feature request related to a problem? Please describe.
I want the Braille table for Māori to appear in the list of input and output Braille tables in the NVDA/Braille settings as it apparently is already added to Liblouis. Ditto for all Liblouis tables not currently visible.

### Describe the solution you'd like
According to [this liblouis page](https://github.com/liblouis/liblouis/releases?page=5), the table for Māori already came out in version 2.6 - along with tables for Afrikaans, Cherokee, Hawaiian, Sotho and Tswan. currently, in the NVDA Braille settings, I see Afrikaans grade 1 and Afrikaans grade 2, but none of the other listed tables. I have also not examined all release notes from other versions to see which other tables are missing. It'd be great if NVDA is able to have access to all Liblouis Braille tables.

### Describe alternatives you've considered
This could be an issue at Liblouis end maybe, if the tables are not in fact present somehow despite the release notes, sorry I don't know how to use the software enough to figure that out so I thought I'd start here.
### Additional context
Using NVDA alpha-30446,d573d133 (2024.2.0.30446)

**Comment by seanbudd** [comment]
Note apparently Maori is not working currently, but should be fixed in an upcoming version of LibLouis: https://groups.io/g/nvda-devel/topic/118337950
